### DISCUSSION TITLE
Can't "star" any projects now in Safari: "This function is not available"

### DISCUSSION THREAD

**Discussion by Stokestack** [community/community#72850] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

If you try to "star" something (and yes I'm logged in), it momentarily does it and then resets. This seems to be a fairly recent bug.

This is in latest Safari (desktop).

Screen grab attached. I've seen the "function not available" message at the bottom of a repo's page, if I remember correctly.

https://github.com/community/community/assets/196304/d7859985-2e97-4a99-8b8f-bc4d94768181

**Comment by ktauchathuranga** [comment]
have you tried using a different browser?

**Reply by InSuperposition** [reply]
This issue still exists in Safari, not Chrome, not Firefox nor Edge

**Comment by Stokestack** [comment]
Thanks for the reply. I haven't, because I'm not interested in changing browsers. But for the sake of troubleshooting...

This bug is not reproducible in Firefox. Title changed accordingly.

**Comment by Neeham** [comment]
Have you tried clearing your Safari browser cache?
Could be the root cause. Try in incognito/private mode :)

**Reply by Stokestack** [reply]
OK, tried both... no dice. Still doesn't "work," in that the status indicator changes back to unstarred. If you refresh, you find that it's starred.

Also, if you press it twice, thinking that it will be toggled back to unstarred and then refresh the page, you'll find that it's still starred.

**Comment by MohamedNourDerbeli** [comment]
You can try some of those : 

Clear Browser Cache:

Sometimes, issues with website functionality can be caused by cached data. Try clearing your browser cache and cookies. After clearing the cache, restart the browser and attempt to star a repository again.

Disable Browser Extensions:

Browser extensions can sometimes interfere with the functionality of websites. Try disabling any browser extensions you have installed, especially ones related to privacy or ad-blocking, and see if the issue persists.

Check Browser Console for Errors:

Open the browser console (usually found in the developer tools) and check for any error messages when you attempt to star a repository. This might provide more information about what's going wrong.

**Reply by Stokestack** [reply]
Thanks. In my reply above I reported no success after clearing the cache and using incognito mode. I have no extensions installed. Just tried checking the log, and there are no errors or warnings.

**Comment by cprinos** [comment]
I see the same problem when using Safari

**Comment by PieterDePauw** [comment]
I am experiencing this problem as well.

**Comment by dbarros** [comment]
I've also been seeing this Github bug in Safari for a while now.
